mgmtfn/k8splugin/kubeClient.go

```diff
@@ -284,9 +284,10 @@ func (c *APIClient) WatchServices(respCh chan SvcWatchResp) {
 				continue
 			}
 
-			//if wss.Object.ObjectMeta.Namespace != "default" {
-			//	continue
-			//}
```

Comment:
If the intention was to just ignore the updates from kube-scheduler and kube-controller-manager, this fix should work and I've verified it. I can add the extra check if you absolutely feel the necessary to do so; but considering it's a temporary fix until we move to the new client-go or figure something out, we can leave it as it is.